### DISCUSSION TITLE
light speed filter agent blocked websites that contain "GitHub.oi" (misspelt so I don't get this blocked)

### DISCUSSION THREAD

**Discussion by TrappedHighschooler** [3kh0/ext-remover#1017] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
how can i open the ways to remove plugins if the plugin blocked it xd

**Comment by Blobby-Boi** [comment]
Use a web view or smth

**Comment by myschoolisass** [comment, marked as answer]
A. its about time, that proxy is so prominent im very surprised they haven't done that yet
B. either bypass lightspeed or find a new provider for game sites (i will not be elaborating on bypassing because i bet almost everyone here is more knowledgable than me)
